Gemfile

```diff
@@ -52,8 +47,9 @@ group :test do
 end
 
 group :development, :test do
-  gem 'byebug', platform: :mri
+  gem 'byebug',             platform: :mri
   gem 'capybara'
+  gem 'database_cleaner-active_record'
```

Comment:
nice one, although you just need it in test env

Reply:
Oh right, thanks for spotting it :)